client/src/lib/changeActiveSubsite.js

```diff
@@ -0,0 +1,20 @@
+import createEvent from 'legacy/createEvent';
+
+export default function changeActiveSubsite(toSubsiteID, toSubsiteName) {
+  const { localStorage } = window;
+  const request = new XMLHttpRequest();
+  const subsiteInfo = {
+    subsiteID: toSubsiteID,
+    subsiteName: toSubsiteName
+  };
+  request.open('GET', `?SubsiteID=${toSubsiteID}`);
```

Comment:
what is this actually doing? will this "refresh the whole CMS"?
is this something that's already done here? https://github.com/silverstripe/silverstripe-subsites/pull/443/files#diff-cf163c29c1a03e751d8439e2f36b5206R33

Reply:
No, it won't update the CMS in terms of what the user sees, but it does "update the CMS" in regards of what the server keeps in session.

The line you referenced (`window.location.search = $.query.set('SubsiteID', subsiteID);`) does reload the CMS in the browser - this by contrast sends an XHR that updates the server and only fires update events on completion, meaning CMS users do not lose their draft.

This file should have this info in a docblock. We'll add one.